2018-11-26-sip-minutes.md

```diff
@@ -0,0 +1,159 @@
+
+[Trait Parameters](https://dotty.epfl.ch/docs/reference/trait-parameters.html)
+
+[Intersection Types](http://dotty.epfl.ch/docs/reference/intersection-types.html)
+
+[Union Types](http://dotty.epfl.ch/docs/reference/union-types.html)
+
+[Dependent Function Types](http://dotty.epfl.ch/docs/reference/dependent-function-types.html)
+
+[Implicit Function Types](http://dotty.epfl.ch/docs/reference/implicit-function-types.html)
+
+[Weak Conformance](https://dotty.epfl.ch/docs/reference/dropped/weak-conformance.html)
+
+[Type Lambdas](https://dotty.epfl.ch/docs/reference/type-lambdas.html)
+
+[Type Checking](https://dotty.epfl.ch/docs/reference/changed/type-checking.html)
+
+[Type Inference](https://dotty.epfl.ch/docs/reference/changed/type-inference.html)
+
+[Implicit Resolution](https://dotty.epfl.ch/docs/reference/changed/implicit-resolution.html)
+
+[Pattern matching](https://dotty.epfl.ch/docs/reference/changed/pattern-matching.html)
+
+[Existential Types](https://dotty.epfl.ch/docs/reference/dropped/existential-types.html)
+
+[Type Projection](https://dotty.epfl.ch/docs/reference/dropped/type-projection.html)
+
+[Class Shadowing](https://dotty.epfl.ch/docs/reference/dropped/class-shadowing.html)
+
+[Trait Parameters](https://dotty.epfl.ch/docs/reference/trait-parameters.html)
+
+
+
+### What’s the plan for keeping the migration period as short as possible?
+A smooth migration process is key for the success of Scala 3. We have learned from our own past experience as well as that of other language communities (for example, Python 3) to recommend the following plan.
+
+These are the key properties of a successful upgrade plan that we recommend to incorporate in the migration to Scala 3.
+
+1. **Incremental.** Instead of a one-time big upgrade, users should be able to adopt Scala 3 at their own pace.
+  
+  a) Compatibility and cross-building. Users should be able to use a common subset of Scala to mix Scala 2 and Scala 3 projects (tbd after the feature freeze) in the same codebase. Scala 2 should guide users towards this subset through deprecation warnings.
+  
+  b) [Tasty](https://www.scala-lang.org/blog/2018/04/30/in-a-nutshell.html) compatibility. Scala 2 and Scala 3 should converge on the use of Tasty, an intermediate representation format, that will have a strong backwards compatibility policy.
+2. **Static types.** Most Scala code leverages static types which makes migrating safer and easier.
+3. **Tooling to ease upgrades.** There should be a rewriting tool for automatic migrations and changes in the current Scala tooling. This tool should accommodate most of the needed changes.
+4. **Shared standard library.** Scala 2.14 and Scala 3.0 should share the same Scala standard library.
+
+In addition, the Scala community has a culture of upgrading the ecosystem on every major version of Scala through the community build. We  recommend to leverage the same mechanism to evaluate Scala upgrades with the migration to Scala 3.
```

Comment:
```suggestion
In addition, the Scala community has a culture of upgrading the ecosystem on every major version of Scala through the community build. We intend to leverage the same mechanism to evaluate Scala upgrades with the migration to Scala 3.
```

Also, the word "evaluate" sounds strange to me here. Is this saying that the migration will be something that Scala users are used to?

Reply:
"intend" is nice.
I also suggest "evaluate" -> "vet". Does that answer your question @hrhino?